common/src/main/java/com/microsoft/identity/common/internal/controllers/LocalMSALController.java

```diff
@@ -94,7 +95,7 @@ public class LocalMSALController extends BaseController {
     private IAuthorizationStrategy mAuthorizationStrategy = null;
 
     @SuppressWarnings({WarningType.rawtype_warning, WarningType.unchecked_warning})
-    private ResultFuture<AuthorizationResult> mAuthorizationResultFuture = null;
+    private Future<AuthorizationResult> mAuthorizationResultFuture = null;
```

Comment:
I've got a PR to merge in the lighter NPE fix into dev, the number of changes will be smaller when merging release/17.1.0 into dev